frontend/packages/dapple/package-post-init.js

```diff
@@ -47,13 +45,14 @@ const tokenSpecs = {
   VSL: { precision: 18, format: '0,0.00[0000000000000000]' },
   PLU: { precision: 18, format: '0,0.00[0000000000000000]' },
   MLN: { precision: 18, format: '0,0.00[0000000000000000]' },
+  RHOC: { precision: 9, format: '0,0.00[0000000]'  }
```

Comment:
You have a supply of 1 billion token. So that means you have 1 Billion units which have X precision decimals. In this case 8. So 1 Token is 1 * 10^8 of minimal unit.
So if we think it in this minimal unit, the total supply would be 1000000000**00000000**